src/building.md

```diff
@@ -233,9 +233,9 @@ To enable package testing during the build process, use instead:
 ```sh
 spack install -v --test=all mfem
 ```
-If you don't have Spack, you can download it and install MFEM with the following commands:
+If you don't have Spack, you can download it and setup your environment with the
+following commands, after which you can install MFEM as above:
 ```sh
 git clone https://github.com/spack/spack.git
-cd spack
-./bin/spack install -v mfem
+. spack/share/spack/setup-env.sh
```

Comment:
This changes the meaning -- the intention was for people to be able to copy and paste and get MFEM built with Spack from scratch.

Maybe that's not a good idea, and we should just tell them where to get Spack from...

I don't think we should include the `. spack/share/spack/setup-env.sh` line though -- it doesn't work for all shells (e.g. I use `tcsh` and it doesn't work for me), and it is not necessary: for a quick build you can just use `./bin/spack`.

Reply:
> Shall I just drop the commit and force push?

Lets do that

Reply:
Done.